base/system/diskpart/diskpart_msg.mc

```diff
@@ -125,26 +125,27 @@ Example:
     ACTIVE
 .
 Language=Romanian
-    On disks with master boot record (MBR) disk formatting, marks
-    the partition with focus as active.
+    Pe discurile cu Înregistrarea de inițializare de maestru (MBR) formatarea
+    discului marchează partiția cu centrare activă.
 
-Syntax:  ACTIVE
+Sintaxă:  ACTIVE
 
-    Writes a value to the disk which is read by the basic input/output
-    system (BIOS) at boot. This value specifies that the partition is
-    a valid system partition.
+    Scrie o valoare pe disc care este citită de sistemul de bază de 
+    intrare/ieșire (BIOS) la inițializare. Această valoare specifică faptul că
+    partiția este o partiție de sistem validă.
 
-    A partition must be selected for this operation to succeed.
+    Pentru a avea succes o partiție trebuie să fie selectată pentru această
```

Comment:
I read this as "for a partition to be successful, it has to be selected for this operation". Please rework.